src/pgduckdb_types.cpp

```diff
@@ -158,6 +160,18 @@ struct DecimalConversionDouble {
 	}
 };
 
+// Util function to convert duckdb `BIT` value to postgres bitstring type.
+// There're two possible corresponding types `BITOID` and `VARBITOID`, here we convert to `VARBITOID` for generality.
+static Datum
+ConvertVarbitDatum(const duckdb::Value &value) {
+	const std::string value_str = value.ToString();
+
+	// Here we rely on postgres conversion function, instead of manual parsing, because BIT string type involves padding
+	// and duckdb/postgres handle it differently, it's non-trivial to memcpy the bits.
+	Datum pg_varbit = pgduckdb::pg::StringToVarbit(value_str.c_str());
+	return pg_varbit;
```

Comment:
```suggestion
	// Here we rely on postgres conversion function, instead of manual parsing, because BIT string type involves padding
	// and duckdb/postgres handle it differently, it's non-trivial to memcpy the bits.
	return pgduckdb::pg::StringToVarbit(value.ToString().c_str());
```